### PR TITLE
[ i18n sprint ] I18n fixes for edit forms

### PR DESCRIPTION
**Pull request by litvinovg**
[Companion PR VIVO-languages](https://github.com/vivo-project/VIVO-languages/pull/116)
[Companion PR Vitro](https://github.com/vivo-project/Vitro/pull/336)
[Companion PR Vitro-languages](https://github.com/vivo-project/Vitro-languages/pull/63)
# What does this pull request do?
This pull request is one of many efforts to improve language neutral Freemarker templates and thus eliminate the need to support language specific templates

Imported fixes from language specific 
https://github.com/vivo-project/VIVO-languages/pull/116/files#diff-bf3f484c537ac9a4842a00b9fd05fa8a5b2736a063d227ea267a4a41a234e1c1
https://github.com/vivo-project/VIVO-languages/pull/116/files#diff-78d57e69e61fc3f97b28ce03f1e1880b2defb2cd999c4c78cdc90c1df272f430
https://github.com/vivo-project/VIVO-languages/pull/116/files#diff-c426a780340d83909740bb200dd8860ac6c52813c1654205c67fe7fff2ada59f

# How to test
* Create a person
* Add role head of 
* Add role memeber of 
* Add advisees 
Create publication
* Add authors via Manage list of authors
* Add editors via Manage list of editors
* Open person profile, click add geographic location, click add new item of this type (leave Campus selected)
You should see something like Create "geographic location" entry for ...

### CODE REVIEW

File: webapp/src/main/webapp/templates/freemarker/edit/forms/addRoleToPersonTwoStage.ftl [hunk 1]
```diff
@@ -195,12 +195,12 @@ Set this flag on the input acUriReceiver where you would like this behavior to o
             <#else>
                 <h4 class="label">${i18n().years_participating} </h4>
                 <#if htmlForElements?keys?seq_contains("startField")>
-                	    <label class="dateTime" for="startField">${i18n().start_capitalized}</label>
+                	    <label class="dateTime" for="startField">${i18n().start_year}</label>
                		    ${htmlForElements["startField"]} ${yearHint}
```
Comment (chenejac) [inline webapp/src/main/webapp/templates/freemarker/edit/forms/addRoleToPersonTwoStage.ftl:199]: This is producing (in adding head of or member of):
Start Year Year 

The word year is two times. The same is for end_year (after adding end_year in the property file)

Comment (chenejac) [reply webapp/src/main/webapp/templates/freemarker/edit/forms/addRoleToPersonTwoStage.ftl:199]: also, a place holder for Organization name has some duplicated text in adding role -  a member of.
Select an existing Select a type or create a new one

Comment (chenejac) [reply webapp/src/main/webapp/templates/freemarker/edit/forms/addRoleToPersonTwoStage.ftl:199]: For Creation of advisor relationship, in the placeholder Advisee and Subject Field are not internationalized

**Review by chenejac — CHANGES_REQUESTED** [review]

**Review by chenejac — APPROVED** [review]
Thanks

**Review by VeljkoMaksimovic — APPROVED** [review]

**Review by VeljkoMaksimovic — APPROVED** [review]

**Review by chenejac — APPROVED** [review]

**Pull request merged**